server/echo_handler_test.cc

```diff
@@ -0,0 +1,25 @@
+#include "gtest/gtest.h"
+#include "httpRequest.h"
+#include "httpResponse.h"
+#include "echo_handler.h"
+#include "../nginx-configparser/config_parser.h"
+#include <vector>
+/*
+classMockNginxConfig : public NginxConfig{ 
+ public:
+  std::vector<std::string> find(const std::string& key) const {
+    return mocked_find(key);
+  }
+  MOCK_CONST_METHOD1(mocked_find,
+*/
```

Comment:
I will also remove this commented code. This was a slightly older version, forgot it was still in here.